### PR TITLE
Create calci.sh

### PR DESCRIPTION
**Pull request by lokeshdayal**
HI team can you please review.

### CODE REVIEW

File: calci.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+3)sum=`expr $n1 \* $n2`
+   echo "Mul = "$sum;;
+
+4)sum=`expr $n1 / $n2`
+    echo "Div = "$sum;;
+
+*)echo "Invalid choice";;
+esac
+
```
Comment (lalasmuathasim) [inline calci.sh:30]: It seems the code will work fine without any bugs
Thanks

File: calci.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+echo "1.Addition"
+echo "2.Subtraction"
+echo "3.Multiplication"
+echo "4.Division"
+
+echo "Enter your choice"
+read ch
+
+case $ch in
+1)sum=`expr $n1 + $n2`
```
Comment (appars) [inline calci.sh:16]: add comments before switch case

**Review by lalasmuathasim — APPROVED** [review]

File: calci.sh (hunk 2 above)
Comment (lokeshdayal) [reply calci.sh:16]: Looks fine this way.

Comment (appars) [reply calci.sh:16]: ok fine

**Review by appars — APPROVED** [review]
code looks good

**Pull request merged**